test/util/pull-request-title.ts

```diff
@@ -224,15 +224,15 @@ describe('PullRequestTitle with custom pullRequestTitlePattern', () => {
 
     it('parses a complex title and pattern', () => {
       const pullRequestTitle = PullRequestTitle.parse(
-        '[HOTFIX] - chore(hotfix/v3.1.0-bug): release 3.1.0-hotfix1 (storage)',
+        '[HOTFIX] - chore(hotfix/v3.1.0-bug): release 3.1.0-hotfix1 (@example/storage)',
```

Comment:
Rather than modifying the existing test (which is fine on its own), please create a new, similar test that verifies that the `@` is an allowed prefix character for component name.